spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/reactive/HttpHandlerAutoConfiguration.java

```diff
@@ -60,8 +61,11 @@ public AnnotationConfig(ApplicationContext applicationContext) {
 		}
 
 		@Bean
-		public HttpHandler httpHandler(ObjectProvider<WebFluxProperties> propsProvider) {
-			HttpHandler httpHandler = WebHttpHandlerBuilder.applicationContext(this.applicationContext).build();
+		public HttpHandler httpHandler(ObjectProvider<WebFluxProperties> propsProvider,
+				ObjectProvider<WebHttpHandlerBuilderCustomizer> handlerBuilderCustomizers) {
+			WebHttpHandlerBuilder handlerBuilder = WebHttpHandlerBuilder.applicationContext(this.applicationContext);
+			handlerBuilderCustomizers.forEach((customizer) -> customizer.customize(handlerBuilder));
```

Comment:
Can you please use `orderedStream()` here so that the customisers will be applied in the order determined by their `@Order` annotation or `Ordered` implementation?

Reply:
Hi @wilkinsona,
I updated the PR and added some documentation.